docs/flashbots-data/mev-inspect-py/architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+title: architecture
+---
+
+#### Classified Traces
+
+For each block we intend to inspect, we first fetch all of its traces, transaction receipts, and other additional information. The raw traces are then processed into classified traces (with protocol name, relevant function signature, relevant call inputs, strategy classification and other information) before being passed onto individual inspectors. 
+
+If we notice these classified traces to only contain liquidation related functions, we can only pass them off to aave/comp inspectors. Similarly, arbitrage profits are reduced by running them through protocol inspectors tagged in this stage (swap/liquidate/buy tx followed by addLiquidity etc). 
```

Comment:
aave/comp inspectors? Strategy inspectors, no?

Reply:
`"Similarly, arbitrage profits are reduced by running them through protocol inspectors tagged in this stage (swap/liquidate/buy tx followed by addLiquidity etc)."`

This is confusing. The protocol inspectors are another stage (what stage is being referred to here?). I don't get what this means either "swap/liquidate/buy tx followed by addLiquidity etc"